[2023-10-13 20:58:46,168] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: consultar_info_covid_api.load_data_redshift scheduled__2023-10-12T00:00:00+00:00 [queued]>
[2023-10-13 20:58:46,175] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: consultar_info_covid_api.load_data_redshift scheduled__2023-10-12T00:00:00+00:00 [queued]>
[2023-10-13 20:58:46,176] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-13 20:58:46,176] {taskinstance.py:1377} INFO - Starting attempt 14 of 19
[2023-10-13 20:58:46,177] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-13 20:58:46,189] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): load_data_redshift> on 2023-10-12 00:00:00+00:00
[2023-10-13 20:58:46,192] {standard_task_runner.py:52} INFO - Started process 192 to run task
[2023-10-13 20:58:46,193] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'consultar_info_covid_api', 'load_data_redshift', 'scheduled__2023-10-12T00:00:00+00:00', '--job-id', '403', '--raw', '--subdir', 'DAGS_FOLDER/api_covid.py', '--cfg-path', '/tmp/tmp0ermdnqn', '--error-file', '/tmp/tmpq40fbrl7']
[2023-10-13 20:58:46,194] {standard_task_runner.py:80} INFO - Job 403: Subtask load_data_redshift
[2023-10-13 20:58:46,235] {task_command.py:371} INFO - Running <TaskInstance: consultar_info_covid_api.load_data_redshift scheduled__2023-10-12T00:00:00+00:00 [running]> on host 738c88afe20a
[2023-10-13 20:58:46,284] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Axel Villegas Luna
AIRFLOW_CTX_DAG_ID=consultar_info_covid_api
AIRFLOW_CTX_TASK_ID=load_data_redshift
AIRFLOW_CTX_EXECUTION_DATE=2023-10-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=14
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-12T00:00:00+00:00
[2023-10-13 20:59:15,878] {logging_mixin.py:115} INFO - Exito
[2023-10-13 20:59:15,879] {python.py:173} INFO - Done. Returned value was: None
[2023-10-13 20:59:15,888] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=consultar_info_covid_api, task_id=load_data_redshift, execution_date=20231012T000000, start_date=20231013T205846, end_date=20231013T205915
[2023-10-13 20:59:15,935] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-13 20:59:15,953] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
